types/convict/convict-tests.ts

```diff
@@ -139,26 +137,28 @@ if (conf.has('key')) {
     conf.set('the.awesome', true);
     conf.load({
         thing: {
-            a: 'b'
-        }
+            a: 'b',
+        },
     });
 }
 
+// @ts-expect-error Trying to access no existing property
 conf.has('unknow.key');
-conf.has<'db', 'ip'>('db.ip');
+conf.has('db.ip');
 
 const schema = conf.getSchema();
 
 const schemaVal = conf.getSchema().properties.db.properties.port.default;
 
 conf.get();
+// @ts-expect-error Trying to access no existing property
 conf.get('unknownkey');
 conf.get('db');
 conf.get('db.ip');
-conf.get<'db', 'ip'>('db.ip');
+conf.get('db.ip');
```

Comment:
If you remove the <...> syntax, then this line is duplicated with the above one. Is this needed?

Reply:
Yes. they are duplicates. I'll have it removed